fairing/kubernetes/manager.py

```diff
@@ -87,7 +107,7 @@ def get_service_external_endpoint(self, name, namespace, selectors=None):
         except client.rest.ApiException as e:
             logger.error("error getting status for {} {}".format(name, str(e)))
 
-    def log(self, name, namespace, selectors=None):
+    def log(self, name, namespace, selectors=None, container='', follow=True):
```

Comment:
container=None?

Reply:
I think that should be "container=''", I tested "container=None", if the container is not defined (you know the parameter is new, and some old callings are not set this), the API prints error say no this container, but "" is OK. Thanks.